browser-support.md

```diff
@@ -1,27 +0,0 @@
----
-title: Browser Support
-page_title: Browser Support
-description: Browser support for Telerik UI for Blazor.
-slug: browser-support
-position: 200
----
-
-# Browser Support
-
-This page lists the browsers supported by Telerik UI for Blazor. You can use these browsers to access web applications built with Telerik UI for Blazor.
-
->caption Browsers supported by Telerik UI for Blazor
-
-| Browser | Supported Versions |
-| ----------- | ----------- |
-| Chrome (including Android and iOS) | Latest version |
-| Edge | Latest version |
-| Firefox | Latest version |
-| Safari (including iOS) | Latest version |
-
-Telerik UI for Blazor steps on the Blazor framework provided by Microsoft. This means that you can host/run the applications on [platforms supported by .NET Core 3](https://docs.microsoft.com/en-us/aspnet/core/blazor/supported-platforms?view=aspnetcore-3.0) - Windows, Linux, MacOS - and use the supported browsers on them.
```

Comment:
I am not sure I understand why we are listing .NET Core 3.0 here regarding supported platforms as we are actually supporting platforms that correspond to different .NET versions (3.1, 6.0, etc.). Furthermore, we are actually supporting .NET Core 3.1 instead of 3.0.

Reply:
This link was outdated and the content has already been removed.